packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte

```diff
@@ -65,11 +65,18 @@
 		}
 		tokenizerConfig = config.tokenizer_config;
 
-		const chatTemplate = tokenizerConfig.chat_template;
+		let chatTemplate = tokenizerConfig.chat_template;
 		if (chatTemplate === undefined) {
 			error = "No chat template found in tokenizer config";
 			return;
 		}
+		if (Array.isArray(chatTemplate)) {
+			chatTemplate = chatTemplate.find((template) => template.name === "default")?.name ?? chatTemplate[0]?.name;
```

Comment:
```suggestion
			chatTemplate = chatTemplate.find((template) => template.name === "default")?.template ?? chatTemplate[0]?.template;
```

IIUC, here we want to filter by name but then retrieve the template, right?